src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt

```diff
@@ -50,12 +51,17 @@ class CosmasServer(port: Int) {
 fun main(args: Array<String>) {
     val arg = CosmasServerArgs(ArgParser(args))
     println("Try to bind in port ${arg.port}")
-    val server = CosmasServer(arg.port)
+    val server =
+            if (arg.bucket != "")
+                CosmasServer(arg.port, CosmasGoogleCloudService(arg.bucket))
+            else
+                CosmasServer(arg.port, CosmasInMemoryService())
     println("Start working in port ${arg.port}")
     server.start()
     server.blockUntilShutDown()
 }
 
 class CosmasServerArgs(parser: ArgParser) {
     val port: Int by parser.storing("--port", help = "choose port") { toInt() }.default { 50051 }
+    val bucket: String by parser.storing("--bucket", help = "choose bucket")

```

Comment:
The same about bucket. `help="GCS bucket name for file storage" ` gives at least an idea that it is not an ice bucket

Reply:
done